### PR TITLE
Fix: retrieve_list pagination doesn't work unless filters  are converted to int

### PR DESCRIPTION
**Pull request by tadams42**

### CODE REVIEW

**Comment by timmartin19** [comment]
Good catch.  Could we use the `ripozo.resource.fields.common.IntegerField` class for the conversion though?  This will fix stuff like the the query args being a dictionary of `unicode: list[unicode]` (which is common in query arg parsers).  Also, a unit test that breaks demonstrates the fix would be awesome to prevent regressions.
Thanks!

**Comment by tadams42** [comment]
I'll add tests for this and let you know when I'm done. 

In the mean time, something else i also broken and I don't think it was me:

```
======================================================================
FAIL: test_get_fields (ripozo_sqlalchemy_tests.integration.columns.TestColumnTypes)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tomislav/dev/personal/ripozo-sqlalchemy/ripozo_sqlalchemy_tests/integration/common.py", line 77, in test_get_fields
    msg='{0} is not an instance of {1}'.format(field, self.field_dict[field]))
AssertionError: <ripozo.resources.fields.common.StringField object at 0x7fb7180b6940> is not an instance of <class 'ripozo.resources.fields.base.BaseField'> : enum is not an instance of <class 'ripozo.resources.fields.base.BaseField'>
----------------------------------------------------------------------
Ran 83 tests in 1.424s
```

This one is failing with or without my commits, and that means this pull request won't be possible to merge.

File: ripozo_sqlalchemy/alchemymanager.py [hunk 1]
```diff
@@ -182,8 +182,13 @@ def retrieve_list(self, session, filters, *args, **kwargs):
         :rtype: list, dict
         """
         query = self.queryset(session)
-        pagination_count = filters.pop(self.pagination_count_query_arg, self.paginate_by)
-        pagination_pk = filters.pop(self.pagination_pk_query_arg, 1)
+        translator = _COLUMN_FIELD_MAP[int]('tmp')
```
Comment (timmartin19) [inline ripozo_sqlalchemy/alchemymanager.py]: I'd rather we use the IntegerField directly.

```
translator = IntegerField('tmp')
```

**Comment by timmartin19** [comment]
@tadamic Yeah, that's due to a change in ripozo proper.  You didn't break anything.  I'll get a fix out for that shortly.

**Comment by tadams42** [comment]
I added integration test for this. It would be to much mocking for unit test and that would just make test code unreadable. Might be cleaner this way, see if it suits you...

**Comment by timmartin19** [comment]
LGTM.  Thanks!

**Pull request merged**